src/IRPrinter.cpp

```diff
@@ -1085,10 +1085,35 @@ void IRPrinter::visit(const Shuffle *op) {
 }
 
 void IRPrinter::visit(const VectorReduce *op) {
+    std::string op_str;
```

Comment:
This should use the existing operator<< for vector reduce op types (with the implementation changed to snake case if you like)

Reply:
I see you changed the implementation of operator<< to use snake case, so I think the changes to this function can be mostly reverted